grafana-plugin/src/containers/DefaultPageLayout/DefaultPageLayout.tsx

```diff
@@ -64,6 +65,9 @@ const DefaultPageLayout: FC<DefaultPageLayoutProps> = observer((props) => {
   const { currentTeam } = teamStore;
   const { currentUser } = userStore;
 
+  const isChatOpsConnected = getIfChatOpsConnected(currentUser);
+  const isPhoneVerified = currentUser?.cloud_connection_status === 3 ? true : currentUser?.verified_phone_number;
```

Comment:
This can be simplified 
`const isPhoneVerified = currentUser?.cloud_connection_status === 3 || currentUser?.verified_phone_number;`

Reply:
It'd also be nice to have the codes defined as constants rather than magical numbers. That could be for future tech debt work.